runtime/node/src/shared/composerBot.ts

```diff
@@ -41,10 +51,14 @@ export class ComposerBot extends ActivityHandler {
     this.projectRoot = getProjectRoot();
     this.settings = getSettings(this.projectRoot);
 
+    ComponentRegistration.add(new AdaptiveComponentRegistration());
+    ComponentRegistration.add(new QnAMakerComponentRegistration());
+    ComponentRegistration.add(new LuisComponentRegistration());
+    // ComponentRegistration.add(new CustomActionComponentRegistration());

```

Comment:
Consider adding inline explaination of what this line is for and when to enable it